tasks/fix-cat2.yml

```diff
@@ -500,10 +501,11 @@
       - sudoers
 
 - name: "MEDIUM | RHEL-07-010350 | PATCH | Users must re-authenticate for privilege escalation."
-  debug:
-    msg: "!authenticate entry for sudo: {{ item }}"
-  with_items: "{{ rhel_07_010350_audit.stdout_lines }}"
-  changed_when: no
+      replace:
+        path: "{{ item }}"
+        regexp: '^([^#].*)!authenticate(.*)'
+        replace: '\1authenticate\2'
```

Comment:
looks good

Reply:
For some reason, this one also grabs `!authenticate` in commented lines, even though what should be an equivalent `NOPASSWD` replace above correctly ignores comments.